### ISSUE TITLE
Handcrafted anchors are renamed when vaulted

### ISSUE THREAD

**Issue by oveee92** [oveee92/vaulti#2]
This works fine, and gets opened in the editor just as it is

```yaml
myvar: &someanchor foo
```

However, this:

```yaml
myvar: &someanchor !ENCRYPT foo
```

results in

```yaml
myvar: &id001 !ENCRYPT foo
```

when opening the editor.

**Comment by oveee92** [comment]
This is a bit of a broader issue, and differs a bit for creating vs editing the variables;

the anchor is actually removed for regular TaggedScalar variables that are returned from custom constructor functions when **creating** variables.

When opening a file with anchors **already present,** it will rename them to the `id00x` format.

**Comment by oveee92** [comment]
This is really two separate issues;
1. the yaml loader will by default discard any unreferenced anchors (fix illustrated in https://stackoverflow.com/a/71920424)
2. and it will rename the referenced ones to an internal reference (fix by setting the anchor value to the TaggedScalar objects before returning them)

**Comment by oveee92** [comment]
Fixed in Version 0.1.3
